### ISSUE TITLE
Is there a service intermittance to verify BNS subdomain data?

### ISSUE THREAD

**Issue by paradigma-cl** [hirosystems/stacks-blockchain-api#1579]
**Describe the bug**
We are detecting that some subdomain names are not found using https://stacks-node-api.mainnet.stacks.co/v1/names/philliproe.xck.app for ie. Some subdomain users are experiencing this issue, and others are working correctly.

**To Reproduce**

**Expected behavior**
It should respond with the zone file contents.  It was working correctly before.

**Screenshots**

**Console log**
If applicable, add console log to help explain your problem.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by rafaelcr** [comment]
I believe this is a duplicate of #1269 which is blocked by a blockchain side bug

**Comment by rafaelcr** [comment]
Closing as duplicate
